packages/react-jsx-highcharts/src/components/Series/Series.js

```diff
@@ -102,8 +88,21 @@ class Series extends Component {
     return config;
   }
 
+  createSeries () {
```

Comment:
Ditto with this method, it should be bound to `this`

```suggestion
  createSeries = () => {
```